sms-authenticator/pom.xml

```diff
@@ -18,6 +18,11 @@
 			<artifactId>keycloak-quarkus-server-app</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.libphonenumber</groupId>
+			<artifactId>libphonenumber</artifactId>
+			<version>8.13.40</version>
```

Comment:
Just a general note: we should probably activate Github Dependabot to automatically bump versions and receive security updates.

Reply:
Solved by 6922721b828f83d23627cbe16993c12a874b906f